spec/models/legal_aid_application_spec.rb

```diff
@@ -1443,6 +1443,25 @@
     end
   end
 
+  describe "#special_children_act_proceedings??" do
```

Comment:
Is this supposed to be a double question mark?

```suggestion
  describe "#special_children_act_proceedings?" do
```

Reply:
🤦‍♂️ merged suggestion!